Translation/en/RedirectedResources/assets/abdata/custom/customscenelist/00/cus_selectlist/translation.txt

```diff
@@ -1,59 +1,59 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.2 (KoikatsuSunshine) by KKS_TextDump v1.4.5.2
 //
-//特殊表現の種類=
-//眉の種類=
-//アイライン上の種類=
-//アイライン下の種類=
-//白目のグラデ種類=
-//ハイライト上の種類=
-//ハイライト下の種類=
-//瞳の種類=
-//瞳のグラデ種類=
-//鼻の種類=
-//リップラインの種類=
-//ホクロの種類=
-//アイシャドウの種類=
-//チークの種類=
-//リップの種類=
-//フェイスペイント０１の種類=
-//フェイスペイント０２の種類=
-//肉感の種類=
-//乳首の種類=
-//陰毛の種類=
-//日焼け跡の種類=
-//ボディーペイント０１の種類=
-//ボディーペイント０２の種類=
-//ボディーペイント０１の配置=
-//ボディーペイント０２の配置=
-//前髪の種類=
-//後ろ髪の種類=
-//横髪の種類=
-//付け毛の種類=
-//トップスの種類=
-//胴の種類=
-//襟の種類=
-//装飾の種類=
-//インナーの種類=
-//アウターの種類=
-//柄の種類=
-//エンブレムの種類=
-//ボトムスの種類=
-//ブラの種類=
-//ショーツの種類=
-//手袋の種類=
-//パンストの種類=
-//靴下の種類=
-//靴の種類=
-//アクセサリ（髪）の種類=
-//アクセサリ（頭）の種類=
-//アクセサリ（顔）の種類=
-//アクセサリ（首）の種類=
-//アクセサリ（胴）の種類=
-//アクセサリ（腰）の種類=
-//アクセサリ（脚）の種類=
-//アクセサリ（腕）の種類=
-//アクセサリ（手）の種類=
-//アクセサリ（股間）の種類=
-//髪のツヤ種類=
-//頭の種類=
+特殊表現の種類=Face Overlay Type
+眉の種類=Eyebrow Type
+アイライン上の種類=Upper Eyeliner Type
+アイライン下の種類=Lower Eyeliner Type
+白目のグラデ種類=Sclera Type
+ハイライト上の種類=Upper Highlight Type
+ハイライト下の種類=Lower Highlight Type
+瞳の種類=Eye Type
+瞳のグラデ種類=Eye Gradient Type
+鼻の種類=Nose Type
+リップラインの種類=Lip Line Type
+ホクロの種類=Mole Type
+アイシャドウの種類=Eyeshadow Type
+チークの種類=Cheek Type
+リップの種類=Lip Type
+フェイスペイント０１の種類=Facepaint 01 Type    //regexe
+フェイスペイント０２の種類=Facepaint 02 Type    //regexe
```

Comment:
You can't add comments in the same line as translation because they will be treated as part of the translation. Add any comments on the line before the translation instead.

Reply:
Removed same line comments for both cus_selectlist files